File: src/main/java/hilfling/backend/hilfing/HilfingApplication.java
package hilfling.backend.hilfing;

import hilfling.backend.hilfing.service.*;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;


@SpringBootApplication
public class HilfingApplication {

    public static void main(String[] args) {
        SpringApplication.run(HilfingApplication.class, args);
    }
}

File: src/main/kotlin/hilfling/backend/hilfling/HilfingApplication.kt
package hilfling.backend.hilfing

import hilfling.backend.hilfling.configurations.FileStorageProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@EnableConfigurationProperties(
    FileStorageProperties::class
)

@SpringBootApplication
open class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
